shoop/core/migrations/0008_maintenance_mode.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shoop', '0007_product_media'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='shop',
+            name='maintenance_mode',
+            field=models.BooleanField(default=True, verbose_name='maintenance mode'),
```

Comment:
This being default=True breaks all tests that need the front middleware to set the login.